lib/ohai/plugins/linux/platform.rb

```diff
@@ -86,6 +86,19 @@ def cumulus_version
   end
 
   #
+  # Determines the platform version for F5 Big-IP systems
+  #
+  # @returns [String] bigip Linux version from /etc/f5-release
+  #
+  def bigip_version
+    release_contents = File.read("/etc/f5-release")
+    release_contents.match(/BIG-IP release (\S*)/)[1] # http://rubular.com/r/O8nlrBVqSb
+  rescue NoMethodError, Errno::ENOENT, Errno::EACCES # rescue regex failure, file missing, or permission denied
```

Comment:
An explanation since it took me a moment: The NoMethodError catches the regex failure because String#match returns `nil` if there was no match and `[]`  isn't a method on `nil`.